content/References/IPv6.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Understanding IPv6"
+description: "Provides information general information on IPv6 and implementation guidance."
+weight: 20
+tags:
+- network
+- interfaces
+- ipv6
+---
+
+## What is IPv6?
+IPv6, referred to as the Internet protocol next generation (IPng), is a 128-bit alphanumeric address organized into eight groups of 16 bits with each group expressed as four-digit hexadecimal numbers separated by colons.
+An example of an IPv6 address is *fe80:cd00:0000:0cde:1257:0000:211e:729c*.
+
+IPv6 addresses are split into two parts: a network (routing prefix) and a node component(interface ID).
+
+The network component is the first 64 bits of the address and is used for routing.
+This component can be split into a block of 48 bits for global network addresses and a block of 16 bits controlled by network administrators for subnets on an internal network.
+
+The node component is the second 64 bits of the IPv6 address and is used to identify the address of the interface.
+It is derived from the physical or MAC address, using the 64-bit extended unique identifier (EUI-64) format defined by the Institute of Electrical and Electronics Engineers (IEEE).
+This format designed to provide the ability to provide unique addresses to any electronic device equipped with sensors and connect them for data sharing.
+
+The IPv6 total address can be shortened. The addressing scheme allows the omission of any leading zero as well as any sequences consisting of only zeros.
+Using the example above, a shorted address looks like *fe80:cd00:0:cde:1257:0:211e:729c*.
+
+The specific layout of IPv6 addresses can vary depending on its format, with three basic parts: routing prefix, subnet ID, and the interface ID.
+The routing prefix and subnet ID represent two main levels in the address which is either global or site specific.
+Internet Registries and Internet service providers (ISPs) can decide to subdivide the routing prefix bits.
+The left most set of numbers or first 48 bits is the site prefix, and the next 16 bits is the subnet ID. The subnet ID lays out site topology.
+The 64 bit interface ID can be manually or automatically configured.
+
+IPv6 provides 1028 times more addresses than IPv4, which is running out of numbers due to the ever increasing number devices world wide wanting to connect.
```

Comment:
IPv6 provides 4294967296 times more addresses than IPv4.

Reply:
Moved this up to the top.